test/integrations/testUtils.ts

```diff
@@ -104,6 +97,42 @@ export const overrideDestination = (destination: Destination, overrideConfigValu
   });
 };
 
+export const produceTestData = (testData: TestCaseData[]) => {
```

Comment:
can we also return the result so that we can use it without reading again from file

Reply:
yeah, returning result would avoid temporary file itself - @utsabc was there any reason to write to file & then read ?

Reply:
Some times these tests data can be lengthy to copy from terminal hence created temp file